recipes/r-imgur/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set version = '1.0.3' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-imgur
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/imguR_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/imguR/imguR_{{ version }}.tar.gz
+  sha256: 6311f971d9a2041250e45c81184d0408f220f37236e62016be7965c41e7cc791
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
```

Comment:
Please add:
```suggestion
  number: 0
  noarch: generic
```

Reply:
@dbast seems that noarch: generic wouldn't work here